src/ui/settings/AudioSettings.tsx

```
@@ -23,23 +23,25 @@ export const AudioSettings = memo(function AudioSettings({
 
+  const handleVolumeChange = (
+    cb: (v: number) => void,
+    e: ChangeEvent<HTMLInputElement> | { target: { value: number } }
+  ) => {
+    const raw = e.target.value
+    const parsed = typeof raw === 'number' ? raw : parseFloat(String(raw))
+    if (!Number.isFinite(parsed)) return
+    cb(parsed)
+  }
+
   const handleMusicChange = useCallback(
```

Biome (2.4.15)

[error] This hook does not specify its dependency on handleVolumeChange.

(lint/correctness/useExhaustiveDependencies)